.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}-latest
+
+    strategy:
+      matrix:
+        # currently windows would be difficult to support due to the build
```

Comment:
This comment is probably unnecessary. I'd suggest omitting it.

Reply:
Done.